packages/cube/test/validation.pipeline.test.js

```diff
@@ -27,7 +27,29 @@ describe('cube validation pipeline', function () {
     const result = shell.exec(command, { silent: true, cwd })
 
     strictEqual(result.code, 1)
-    expect(result.stderr).to.match(/^Error: At least 1 violations found/)
+    expect(result.stderr).to.match(/1 violations found/)
+    ok(result.stdout.includes('_:report <http://www.w3.org/ns/shacl#conforms> "false"^^<http://www.w3.org/2001/XMLSchema#boolean>'))
+  })
+
+  it('should run check-cube-observations pipeline without error when maxViolations is not exceeded', () => {
```

Comment:
I'm not convinced this behaviour is useful: the main purpose of maxViolation is to avoid bloating the output (usually a few examples are enough to understand what's wrong with the input) but the cli should succeed only if there are no violations at all, allowing to break a script in which the validation pipeline is used to ensure that all the input is conformant

Reply:
You're right. Will change that. Then maybe we need a much higher default. Like 100 or more?